applicants/README.md

```diff
@@ -0,0 +1,154 @@
+# applicants
+
+## applicants object
+
+```{
+  userId: string,
+  biodata: {
+    firstName: string,
+    lastName: string,
+  },
+  location: {
+    city: string,
+    state: string,
+    country: string,
+  },
+  professional: {
+    institution: string,
+    skills: string,
+  },
+  intro: {
+    introduction: string,
+    funFact: string,
+    forFun: string,
+    whyRds: string,
+    numberOfHours: string,
+  },
+  foundFrom: string,
+  status: string || null,
+  discordLink: string || null
+}
+```
+
+## **Requests**
+
+|                         Route                          |             Description              |
+| :----------------------------------------------------: | :----------------------------------: |
+|                [GET /applications](#get-applications)                |   Returns all the applications in the system   |
+|           [GET /applications/:userId](#get-applicationsuserid)            | Retruns application of a particular user |
+|  [POST /applications](#post-applications)   |    Add application of a user    |
+|       [PATCH /applications/:userId](#patch-applicationsuserid)       |   Updates application of a user   |
+|        |
+
+## **GET /applications/**
+
+Return all the applications which are not accepted or rejected, this API will only be accessible to super_user
+
+- **Params**  
+  None
+- **Query** 
+  None
+- **Body**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
```

Comment:
Shouldn't this be a super user-only route to fetch all applications?

cookie should be SUPER_USER_JWT (probably)

Reply:
it will not be super_user_jwt